client/app/src/androidTest/java/com/example/mrides/CreateRouteUserInterfaceTest.java

```diff
@@ -25,7 +27,12 @@
 
 
 @RunWith(AndroidJUnit4.class)
-public class CreateRouteUserInterfaceTest {
+public class CreateRouteUserInterfaceTest extends ActivityInstrumentationTestCase2<CreateRouteActivity> {
+
+    public CreateRouteUserInterfaceTest()
+    {
```

Comment:
Opening curly bracket should be in the same line and at the end of the constructor